packages/data-point/lib/entity-types/entity-request/resolve.js

```diff
@@ -106,7 +106,31 @@ module.exports.resolveBeforeRequest = resolveBeforeRequest
 
 function resolveRequest (acc, resolveTransform) {
   inspect(acc)
-  return rp(acc.options).then(result => utils.set(acc, 'value', result))
+  return rp(acc.options)
+    .then(result => utils.set(acc, 'value', result))
+    .catch(error => {
+      const message = [
+        'Entity info:',
+        '\n  Id: ',
+        _.get(acc, 'reducer.spec.id'),
+        '\n',
+        utils.inspectProperties(acc, ['options', 'params', 'value'], '  '),
+        '\n  Request:\n',
+        utils.inspectProperties(
+          error,
+          ['error', 'message', 'statusCode', 'options', 'body'],
+          '  '
+        )
+      ].join('')
```

Comment:
This would be easier to read as a template literal.

Reply:
tried both, on all of the long error messages we have we been following this pattern, would rather stick to the pattern. 

With template literals there is a point where it becomes really hard to read, specially after prettier does its 'thing'

Reply:
OK, fair enough.